### ISSUE TITLE
Conventions / Guidelines

### ISSUE THREAD

**Issue by KevinTss** [HackYourFutureBelgium/improvements#3]
As developer you know that conventions are so important, and we need to be an example for students. 
Is for me mandatory that we define somewhere some guideline about conventions. 

WHY?
Because we tach that to our students
Because for the moment there are lot of inconsistencies:
- Example these repo names are in CamelCase: https://github.com/HackYourFutureBelgium/JavaScript3
https://github.com/HackYourFutureBelgium/CommandLine
- This one in snake_case:
https://github.com/HackYourFutureBelgium/teaching_tips_tricks
- This one in kebab-case:
https://github.com/HackYourFutureBelgium/built-with-branches
- This one... we don't know:
https://github.com/HackYourFutureBelgium/NodeJS_course

We need a way to make that more consistent, is the base of the development.

**Comment by fdb** [comment]
Good point! I normally avoid uppercase letters in my repository names, and use dashes to split words. That's also how [Facebook](https://github.com/facebook/) and [Google](https://github.com/google/) do it.

**Comment by colevandersWands** [comment]
maybe if this were the readme in an "awesome-conventions" repo? then there's a single reference point of HYF conventions that can be covered in class as they are relevant

**Comment by colevandersWands** [comment]
or would the repo be called "Aw3s0me_conventions"

**Comment by KevinTss** [comment]
Not sure about create one repo for that, is probably a bit over engineer for that. But a guideline repo with a how to navigate across all repositories and convention, where find what informations,...
For me that could be better. It could be kind of general documentation of HYF belgium organisation

**Comment by colevandersWands** [comment]
a new home page for the organization, work in progress: https://home.hackyourfuture.be.

and a contributors repo that will have the contribution guidelines (guidelines not yet written): https://github.com/hackyourfuturebelgium/contributors
